plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -7529,40 +8097,55 @@ table.bar_chart {
 	}
 }
 
+@media only screen and (max-width: 1280px) {
+	#woocommerce-product-data,
+	#woocommerce-coupon-data {
+		.options_group,
+		// General coupos data does not have options group wrapper, make it look like it has.
+		&#general_coupon_data {
```

Comment:
The compiled selector `#woocommerce-coupon-data#general_coupon_data` seems to be incorrect.

![image](https://github.com/user-attachments/assets/e20ca80b-a31d-4d24-a62f-2364c3b0692c)